installer/vic-unified-installer.mk

```diff
@@ -0,0 +1,38 @@
+# Copyright 2016 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+PACKER ?= packer
+OVFTOOL ?= ovftool
+
+PHOTON_ISO := https://bintray.com/vmware/photon/download_file?file_path=photon-1.0-62c543d.iso
+PHOTON_ISO_SHA1SUM := c4c6cb94c261b162e7dac60fdffa96ddb5836d66
+
+.PHONY: ova-release
+
+ovfenv := $(BIN)/ovfenv
+ovfenv: $(ovfenv)
+
+$(ovfenv): $$(call godeps,installer/ovatools/ovfenv/*.go)
+	@echo building ovfenv linux...
+	@GOARCH=amd64 GOOS=linux $(TIME) $(GO) build $(RACE) -ldflags "$(ldflags)" -o ./$@ ./$(dir $<)
+
+ova-release: $(ovfenv)
+	@echo building vic-unified-installer OVA using packer...
+	@cd $(BASE_DIR)installer/packer && $(PACKER) build \
```

Comment:
I understand there is still more work during the coming sprint, but do we need any unit test added as part of the unified installer? Otherwise looks good to me.